src/routes/categories.svelte

```diff
@@ -2,26 +2,46 @@
   import BasePage from "./../ui/BasePage.svelte";
 
   import Header from "./../ui/Header.svelte";
+  import Map from "../ui/map/Map.svelte";
   import TopicExplorer from "./../ui/TopicExplorer.svelte";
+  import Topic from "../ui/Topic.svelte";
   import Feedback from "./../ui/Feedback.svelte";
+
+  import { selectedGeography } from "../model/geography/geography";
+
+  let englandWalesBounds = [2.08, 55.68, -6.59, 48.53];
+
+  $: innerWidth = 0;
 </script>
 
+<svelte:window bind:innerWidth />
 <svelte:head>
   <title>2021 Census Data Atlas Categories</title>
   <script defer src="https://cdn.ons.gov.uk/sdc/design-system/44.1.2/scripts/main.js"></script>
 </svelte:head>
 
-<BasePage>
+<BasePage mobileMap={false} withoutBackground>
   <span slot="header">
     <Header
       showBackLink
       serviceTitle="Choose a data option"
-      description="Choose a category and select an option within it to explore Census data."
+      description="Choose a category and select an option within it to explore {$selectedGeography.lad
+        ? `${$selectedGeography.lad}'s`
+        : 'Census'} data."
     />
   </span>
 
+  <span slot="map">
+    <Map bounds={englandWalesBounds} />
```

Comment:
Instead of setting `englandWalesBounds` across a couple of components, can we just make them a const in the `<Map />` and use them as default?

Reply:
We already have in the `<Map />` component a default value for the `bounds` prop, the reason why we are passing down a new set of coordinates is because the map centres different on different screen size.